lib/NMISNG/Outage.pm

```diff
@@ -561,13 +561,27 @@ sub check_outages
 
 				for my $propname (keys %{$maybeout->{selector}->{$selcat}})
 				{
-					my $actual = ($selcat eq "config"?
+					my $actual;
+					if ($propname =~ /configuration/){
+						my $config_propname = $propname;
+						$config_propname =~ s/configuration.//;
+						$actual = ($selcat eq "config"?
+												$globalconfig->{$config_propname} :
```

Comment:
Please refactor this so it's not a giant ternary, if statements please so that it's more readable

Reply:
updated the giant ternary to if else